### ISSUE TITLE
Write to composer.json using CLI

### ISSUE THREAD

**Issue by tomasfejfar** [composer/composer#7208]
**When I run this command:**

```
composer config scripts.phpcs "phpcs -n --extension=php"
```

**I get the following output:**

```
[InvalidArgumentException]
  Setting scripts.phpcs does not exist or is not supported by this command
```

**And I expected this to happen:**
It will add "phpcs" script to composer.json that launches phpcs. This would be a great feature to programatically add tooling to composer scripts.

**Comment by Seldaek** [comment]
Fixed by #7225

**Comment by tomasfejfar** [comment]
Great, thanks
